NEWS.rst

```diff
@@ -40,6 +40,7 @@ Lua
 New Features
 ------------
 - `gui/gm-editor` it's now possible to insert default types to containers. For primitive types leave the type entry empty, for references use ``*``.
+- `dwarfvet` enables animal caretaking.
```

Comment:
Should be under "new plugins" (which probably doesn't exist, but see the initial comment or previous releases for where it goes).